src/components/pages/Room/style.css

```diff
@@ -0,0 +1,52 @@
+/* .link {
+  text-decoration: none;
+  color: rgb(26, 26, 26);
+} */

```

Comment:
- [ ]  Please, remove this old piece of commented code in order to improve the repo's cleanliness  👍